manifest.json

```diff
@@ -14,31 +15,25 @@
     "requirements": {
         "yunohost": ">=2.4.0"
```

Comment:
Have a look to the minimum version required for the helper you want to use.
https://github.com/YunoHost/yunohost/pull/664
Just saying that because that's quite an old version of YunoHost.

Reply:
updated to 3.4